src/OpenTelemetry.Contrib.Shared/OpenTelemetry.Contrib.Shared.csproj

```diff
@@ -5,8 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Api" Version="1.2.0" />
-    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
+    <PackageReference Include="OpenTelemetry.Api" Version="1.4.0" />
```

Comment:
This could be updated to `$OpenTelemetryCoreLatestVersion` after #1042 is merged.